aiida_quantumespresso/parsers/raw_parser_cp.py

```diff
@@ -210,11 +188,12 @@ def parse_cp_raw_output(out_file,xml_file=None,xml_counter_file=None):
             raise AssertionError('%s found in both dictionaries' % key)
         # out_data keys take precedence and overwrite xml_data keys,
         # if the same key name is shared by both (but this should not happen!)
+
     final_data = dict(list(xml_data.items()) + list(out_data.items()) + list(xml_counter_data.items()))
 
     # TODO: parse the trajectory and save them in a reasonable format
 
-    return final_data,job_successful
+    return final_data, job_successful
```

Comment:
This was the old signature of return value for parsers, a tuple of a list of output nodes and a boolean, indicating success or failure. Since now we have the `self.out` and `self.exit_codes` respectively, we no longer need to return anything here

Reply:
Scratch that, this is the raw parser, where it is fine.